src/librustdoc/html/static/main.js

```diff
@@ -223,8 +223,38 @@
             }
         }
     }
+
+    function expandSection() {
+        var hash = getPageId();
+        if (hash === null) {
+            return;
+        }
+
+        var elem = document.getElementById(hash);
+        if (elem && isHidden(elem.offsetParent)) {
```

Comment:
`offsetParent` can be `null`, please check it before calling `isHidden`.

Reply:
Ditto